can-list.js

```diff
@@ -205,6 +205,16 @@ var List = Map.extend(
 		serialize: function () {
 			return mapHelpers.serialize(this, 'serialize', []);
 		},
+		// Function that serializes the passed arg if
+		// type does not match MapType of `this` list
+		// then adds to args array
+		serializeNonTypes: function(MapType, arg, args) {
```

Comment:
We shouldn't add unnecessary things to the prototype.  Please make this a function in the surrounding scope.